main.go

```diff
@@ -204,19 +303,20 @@ func execJQProgram(program, path string, jsonBytes []byte) ([]*jq.Jv, error) {
 
 	fileJv, err := jq.JvFromJSONBytes(jsonBytes)
 	if err != nil {
-		panic("failed to convert jsonified file into jv")
+		return nil, fmt.Errorf("unable to convert to json value from bytes: %s", err)
```

Comment:
@jzelinskie this was swallowing ther error before and also seems like the errors are in fact somewhat useful when returned even. Was thinking potentially could add the jsonBytes to the error conditionally (perhaps in a verbose mode).